sails/api/controllers/polls/delete.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
```

Comment:
Redundant. Poll should be automatically deleted via posts/delete.js, or inside DB on cascade